ssl/statem/statem_lib.c

```diff
@@ -575,6 +575,11 @@ MSG_PROCESS_RETURN tls_process_cert_verify(SSL_CONNECTION *s, PACKET *pkt)
         }
     } else {
         j = EVP_DigestVerify(mctx, data, len, hdata, hdatalen);
+#ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
+        /* Ignore bad signatures when fuzzing */
+        if (SSL_IS_QUIC_HANDSHAKE(s))
+            j = 1;
+#endif
```

Comment:
If we are deepening the role of `FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION` I wonder if we should split it out a bit, given that the asan/ubsan CI also uses this. We need this for fuzzing builds but we don't need this sort of thing, the memsets, etc. for the asan/ubsan builds.

I'd consider doing this &mdash; I'd like the asan/ubsan builds to be as 'normal' as possible.

Reply:
There is no point in having FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION for asan/ubsan builds. And matt already dropped these in a separate commit.